dashboard/config/scripts/circuit_playground_map_buzzer.external

```diff
@@ -1,5 +1,5 @@
 name 'Circuit Playground Map: Buzzer'
 display_name 'Circuit Playground: Buzzer'
 markdown <<MARKDOWN
-<object data="https://docs.code.org/csd/maker_buzzer/" type="text/html"  style="width: 100%; height: 700px;"></object>
+<object data="https://studio.code.org/docs/csd/maker_buzzer/" type="text/html"  style="width: 100%; height: 700px;"></object>
```

Comment:
I noticed that the content at https://studio.code.org/docs/csd/maker_buzzer/ contains a link to https://docs.code.org/applab/buzzer/ . not sure if this is already in your list, but seems like one more place we'll have to make sure gets updated, not right now but whenever we tackle updating CB content

Reply:
I think this change will fix that - https://github.com/code-dot-org/curriculumbuilder/pull/219

Reply:
good point -- I didn't realize this was just a result of `get_absolute_url` being invoked, but that makes sense now.